extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -52,25 +53,32 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
     debouncedMeasurementChangeTimestamp,
   ]);
 
-  // ~~ DisplayStudySummary
-  useEffect(() => {
```

Comment:
State check here is fancy, but can probably also key off of if `trackedStudy` is an empty string or not.